src/Access/UsersConfigAccessStorage.cpp

```diff
@@ -895,7 +895,8 @@ void UsersConfigAccessStorage::load(
             Settings::checkNoSettingNamesAtTopLevel(*new_config, users_config_path);
             parseFromConfig(*new_config);
             access_control.getChangesNotifier().sendNotifications();
-        });
+        },
+        /* load_encryption_codecs_ */ false);
```

Comment:
Maybe it's better to define enum classes:

```cpp
enum class LoadEncryptionCoded
{
    Yes,
    No
};
```

That would force to write in l. 899 `LoadEncryptionCodecs::No` and we would not need to inline comment above ^^.

Reply:
Seem my other comment in ConfigProcessor.h ... it is unexpected to pass something about loading encryption codecs into `ConfigReloader`. Why would we _not_ want to reload encryption codecs? What if the user updates the config file after a while and defines completely different codecs?

Reply:
> Seem my other comment in ConfigProcessor.h ... it is unexpected to pass something about loading encryption codecs into `ConfigReloader`. Why would we _not_ want to reload encryption codecs? What if the user updates the config file after a while and defines completely different codecs?

We'd always want to reload encryption codecs here, since they are defined in config.xml, that's what is done here.